charts/cluster-api-runtime-extensions-nutanix/templates/ccm/aws/manifests/helm-addon-installation.yaml

```diff
@@ -31,4 +29,8 @@ data:
     {{ "{{" }}$ccmVersion := get $k8sMinorVersionToCCMVersion ( print $clusterSemver.Major "." $clusterSemver.Minor ) {{ "}}" }}
     image:
       tag: {{ "{{ " }} $ccmVersion {{ "}}" }}
```

Comment:
Instead of this can we please use Helm import files instead of embedding here? I think this will also simplify the sync tooling as you wouldn't need to parse the templates in the same way, or escape the templated values in the templates. That would mean syncing the actual values templates to `charts/cluster-api-runtime-extensions/addon-values` (or something like that) and including it like:

```
    {{ .Files.Get "addon-values/aws-ccm-values.yaml" }}
```

Reply:
I've tried this approach and its not working well with `helm template`

I think we're going to need to do this parsing anyways because I'm getting information like the name configmap as well as the `if` pipeline that was previously there.